src/npm.js

```diff
@@ -27,14 +27,15 @@ class Workspace {
   */
   async extract(url, target) {
     debug('extract', { url, target });
-    let path = fsPath.join(this.dir, 'node_modules.tar.gz');
+    let path = fsPath.join(target, 'node_modules.tar.gz');
 
-    let req = download({ extract: true }).
+    let req = download().
       get(url).
       dest(target).
-      use(downloadStatus());
+      use(downloadStatus())
 
     await denodeify(req.run).call(req);
+    await run('tar', ['zxf', path], { cwd: this.dir });
```

Comment:
This is platform specific if we need to do this can you make a comment describing why? (and file a followup so we can keep this platform agnostic)